src/lib/Types/Source.hs

```diff
@@ -307,13 +309,18 @@ data SourceBlock' =
  | Command CmdName (UExpr VoidS)
  | DeclareForeign SourceName (UAnnBinder AtomNameC VoidS VoidS)
  | DeclareCustomLinearization SourceName SymbolicZeros (UExpr VoidS)
- | GetNameType SourceName
- | ImportModule ModuleSourceName
- | QueryEnv EnvQuery
- | ProseBlock Text
- | CommentLine
- | EmptyLines
+ | Misc SourceBlockMisc
  | UnParseable ReachedEOF String
+ | BadSyntax Errs
+  deriving (Show, Generic)
+
+data SourceBlockMisc
```

Comment:
Not sure if I'm a fan of this misc category. How do we decide what goes in there?

Reply:
This is the stuff that requires no concrete->abstract syntax interpretation (and isn't a parse error).  Comment added.